src/infill.cpp

```diff
@@ -428,7 +426,8 @@ void Infill::generateCubicSubDivInfill(Polygons& result, const SliceMeshStorage&
 {
     Polygons uncropped;
     mesh.base_subdiv_cube->generateSubdivisionLines(z, uncropped);
-    result = uncropped.cut(outer_contour.offset(infill_overlap));
+    constexpr bool restitch = false; // cubic subdivision lines are always single line segments - not polylines consisting of multiple segments.
+    Polygons polylines = outer_contour.offset(infill_overlap).intersectionPolyLines(uncropped, restitch);
```

Comment:
`result` polygon is not assigned.

Reply:
Fixed in https://github.com/Ultimaker/CuraEngine/commit/bfe73e6e231efd0e18917c2a85cc6348cd7b849f

I tested the change and I pushed to master directly.